torchbox/wagtail-torchbox
# Theme fixes for proposition pages (and elsewhere

Theme fixes for proposition pages (and elsewhere), and remove the option to add the dark transparent theme for proposition pages

- Ticket: https://torchbox.monday.com/boards/1192293412/views/4019870/pulses/1193036756?term=image
- I have not followed a design for this

Overview of changes
- Removes the slightly crude overrides of theme colours that were previously done for all proposition pages, and instead adds some extra theme variables to handle the fact that the dark theme has links that sit both on the dark header area, and on the white body of the page.
- Removes the option to set a proposition page with the 'dark transparent' theme, as this has never worked.

The main aim of this ticket was to ensure that the 'dark' theme used the light blue colours for accents - see this preview of an unpublished service page on the live site to see how this previously looked for service pages: https://torchbox.com/admin/pages/1196/edit/preview/?mode=

In testing though I found a number of issues with different components in different themes which I have addressed as part of this ticket.

To test:
The pattern library page (http://localhost:8000/pattern-library/render-pattern/patterns/pages/proposition/proposition.html) uses the light theme, and does not show all the elements, so it is best to create a new wagtail proposition page (or edit an existing one), and test the following items:
- Links in the intro field (in the hero) for all themes (these have changed a bit in order to create a hover effect)
- In page nav links in the hero and sticky header in all themes
- Links in intro fields for 'our services', 'our clients' etc
- Key points headings in all themes
- CTA without an embed in all themes, with a span tag to change the colour
- Check the hover state on the CTA button doesn't add a pale blue bottom border
- CTA *with* an embed in all themes, with a span tag to change the colour
- Check the hover state on the CTA with embed button doesn't add a pale blue bottom border
